.github/actions/repo-gardening/src/tasks/check-description/index.js

```diff
@@ -0,0 +1,305 @@
+/**
+ * External dependencies
+ */
+const moment = require( 'moment' );
+
+/**
+ * Internal dependencies
+ */
+const debug = require( '../../debug' );
+const getLabels = require( '../../get-labels' );
+const getNextValidMilestone = require( '../../get-next-valid-milestone' );
+const getPluginNames = require( '../../get-plugin-names' );
+
+/* global GitHub, WebhookPayloadPullRequest */
+
+/**
+ * Check if a PR has unverified commits.
+ *
+ * @param {GitHub} octokit - Initialized Octokit REST client.
+ * @param {string} owner   - Repository owner.
+ * @param {string} repo    - Repository name.
+ * @param {string} number  - PR number.
+ *
+ * @returns {Promise<boolean>} Promise resolving to boolean.
+ */
+async function hasUnverifiedCommit( octokit, owner, repo, number ) {
+	let isUnverified = false;
+
+	for await ( const response of octokit.paginate.iterator( octokit.pulls.listCommits, {
+		owner: owner.login,
+		repo,
+		pull_number: +number,
+	} ) ) {
+		isUnverified = !! response.data.find( commit =>
+			commit.commit.message.includes( '[not verified]' )
+		);
+	}
+
+	return isUnverified;
```

Comment:
As written, this will only check if the last "page" of commits has `[not verified]`. We probably want to check all the commits.
```suggestion
	for await ( const response of octokit.paginate.iterator( octokit.pulls.listCommits, {
		owner: owner.login,
		repo,
		pull_number: +number,
	} ) ) {
		if ( response.data.find( commit =>
			commit.commit.message.includes( '[not verified]' )
		) ) {
			return true;
		}
	}

	return false;
```